LanguageApp/app/screens/AboutScreen.js

```diff
@@ -0,0 +1,68 @@
+import { StyleSheet, View, ScrollView } from "react-native";
+import React from "react";
+
+import AppTitle from "../components/AppTitle";
+import Screen from "../components/Screen";
+import AppButton from "../components/AppButton";
+import routes from "../navigation/routes";
+import colors from "../config/colors";
+import AppText from "../components/AppText";
+
+function UserTypeScreen({ navigation }) {
+  return (
+    <Screen>
+      <View style={styles.container}>
+        <AppTitle>What is Locute?</AppTitle>
+        <ScrollView
+          style={styles.scrollView}
+          contentContainerStyle={styles.contentContainer}
+        >
+          <AppText>
+            With this app, you can study as a Language Learner or upload
+            translations of prompts and answers as a Content Provider.{"\n"}
+            {"\n"}As a Language Learner, you can browse according to Language
+            and Category to choose a scenario you would like to study. Play the
+            prompt recording to listen to the prompt spoken aloud. You can then
+            choose to record your response, which will be transliterated, or you
+            may simply type in your answer. When you submit, your answer will be
+            graded compared to the Content Provider's stored response.{"\n"}
+            {"\n"}As a Content Provider, you will choose your language and see
+            which categories and scenarios need translations. You will record
+            and enter text of your translation of prompts and answers. Your
+            reponses will be saved for Language Learners to study from.{"\n"}
```

Comment:
I think you can just separate with different `<AppText>` segments. Kind of like `<p>` in HTML.

This is fine for now, you can adjust later if you'd like.